src/table/column-filter.scss

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-column-filter {
+  @include iui-dropdown;
+  padding: $iui-baseline $iui-sm;
+  width: fit-content;
+
+  .iui-overlay {
+    position: relative;
+
+    > .iui-input {
+      padding-right: $iui-l * 2;
+    }
+
+    > .iui-borderless {
+      position: absolute;
+      right: 0;
+    }
+  }
+
+  .iui-right-align {
```

Comment:
What happens visually without this class? From the name, it seems like this class should not be explicitly needed.

Reply:
Without `iui-right-align`:
![Screen Shot 2021-04-28 at 3 06 36 PM](https://user-images.githubusercontent.com/849817/116459099-3e228d80-a81a-11eb-92f1-537c7ac668c5.png)

Reply:
Right, so we never not want to right alone. I would just make it default behavior that does not need an extra class.

Reply:
The toggle switch is still left aligned though.

Reply:
How does that change anything? You can just right align all `.iui-input-container` elements

Reply:
Yeah, seems like everything should be aligned to the right be default.

Reply:
Latest push right aligns everything & we explicitly tell the toggle switch to be left aligned.